pymedphys/_experimental/pseudonymisation/__init__.py

```diff
@@ -11,12 +11,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 import functools
 import json
 import logging
 from os.path import abspath, basename, dirname, isdir, isfile
 from os.path import join as pjoin
 
+from immutables import Map as frozenmap
```

Comment:
Yup, key is running:

```
poetry add immutables
```

Then, within pyproject.toml change the version number to * and add optional like the others.

Then, within here we actually need to import it with `from pymedphys._imports import immutables`.

And lastly, add immutables to the following file:

https://github.com/pymedphys/pymedphys/blob/8b2b88f3659d7df3907dc17ff378987f2df2c493/pymedphys/_imports/imports.py#L6